index.html

```diff
@@ -0,0 +1,191 @@
+                        <div class="section5-p">
+                            <p>Hello I’m a software developer! I can help <br> you build a product , feature or website <br> Look through some of my work and <br> experience! If you like what you see and <br> have a project you need coded, don’t <br> hestiate to contact me.</p>
+                        </div>
+                        <h3 class="header">LET'S CONNECT</h3>
+                    <div class="icons-container">
+                        <ul>
+                        <li><a href=""><img class="social-icons" src="images/twitter.png" alt=""></a></li>
+                        <li><a href=""><img class="social-icons" src="images/medium.png" alt=""></a></li>
+                        <li><a href=""><img class="social-icons" src="images/github.png" alt=""></a></li>
+                        <li><a href=""><img class="social-icons" src="images/angelist.png" alt=""></a></li>
+                        <li><a href=""><img class="social-icons" src="images/angellist.png" alt=""></a></li>
+                    </ul>
+                    </div>
+                <div class="resume">
+                    <img src="images/resume.png" alt="">
+                </div>
+                              
+                <div class="section5-head">
+                    <h1>Languages</h1>
+                </div>
+                <div class="lang">
+                    <img class="lango" src="images/down-arrow.png" alt="">
+                </div>  
+                <div class="languages">
+                    <div class="language-icon1">
+                        <img src="images/js.png" alt="">
+                        <a href=""><h4 class="icon1-head">javascript</h4></a>
+                    </div>
+                    <div class="language-icon2">
+                        <img src="images/html-new.png" alt="">
+                        <a href=""><h4 class="icon1-head">HTML</h4></a>
+                    </div>
+                    <div class="language-icon3">
+                        <img src="images/css-new.png" alt="">
+                        <a href=""><h4 class="icon1-head">CSS</h4></a>
+                    </div>
+                </div>   
+                <div class="section5-head2">
+                    <h1>Frameworks</h1>
+                    <img class="arrow-next1" src= "images/next.png" alt="">
+                </div>
+                <div class="section5-head3">
+                    <h1>Skills</h1>
+                    <img class="arrow-next2" src="images/next.png" alt="">
+                </div>
+
+<section class="first-section">
+    <h2>Contact Me</h2>
+    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    <form action="https://formspree.io/f/mjvzkobj" method="post">
+        <input type="text" required placeholder="Alexendra loremipsum" class="name" minlength="30" maxlength="30">
```

Comment:
- [ ] the HTML validation required for the name field is on `required` and `maxlength="30"` kindly remove the `minlength="30"` because its not in the requirements and its not logic to have a name with minimum 30 characters

Reply:
Good job following the advice of the previous reviewer